src/NexusMods.App/AppConfig.cs

```diff
@@ -64,15 +64,13 @@ public void Sanitize(IFileSystem fs)
 
 public interface ILoggingSettings
 {
-    /// <summary>
-    /// Path where the log files will be saved to.
-    /// </summary>
-    public ConfigurationPath FilePath { get; }
+    public ConfigurationPath MainProcessLogFilePath { get; }
 
-    /// <summary>
-    /// Path to historical log files, with templated element e.g. 'PATH_TO_FILE/nexusmods.app.{##}.log'
-    /// </summary>
-    public ConfigurationPath ArchiveFilePath { get; }
+    public ConfigurationPath MainProcessArchiveFilePath { get; }
+
+    public ConfigurationPath SlimProcessLogFilePath { get; }
+
+    public ConfigurationPath SlimProcessArchiveFilePath { get; }
```

Comment:
Missing docs

Reply:
Do you really need/want docs for this?

Reply:
`ArchiveFilePath` might not be immediate, some explanation for what is SlimProcess might be useful as well